src/viser/client/src/App.tsx

```diff
@@ -77,7 +78,15 @@ export type ViewerContextContents = {
     "ready" | "triggered" | "pause" | "in_progress"
   >;
   getRenderRequest: React.MutableRefObject<null | GetRenderRequestMessage>;
-  sceneClickEnable: React.MutableRefObject<boolean>;
+  // Track click drag events.
+  sceneClickInfo: React.MutableRefObject<{
+    enabled_click: boolean; // Enable click events.
+    enabled_box: boolean; // Enable box events.
```

Comment:
does it make sense to replace this with something like `enabled: false | "click" | "rect-select"` ?